projects/packages/jetpack-mu-wpcom/src/features/wp-admin-locale-sync/wp-admin-locale-sync.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * Sync wp-admin locale with Calypso locale.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+
+if ( function_exists( 'wpcom_is_nav_redesign_enabled' ) && wpcom_is_nav_redesign_enabled() && $_SERVER['REQUEST_METHOD'] !== 'POST' ) {
+	add_filter(
+		'admin_init',
+		function () {
+			// If we are on the profile update page.
```

Comment:
If we want to do something on `/wp-admin/profile.php`, we can add an action to `load-profile.php` hook. Example:

https://github.com/Automattic/jetpack/blob/51cc27773010e7a0e3ff5ce10a0c5e9598a77791/projects/plugins/jetpack/modules/masterbar/profile-edit/bootstrap.php#L22

Reply:
We actually want to NOT do something (sync) when `/wp-admin/profile.php` POSTs and redirects to `/wp-admin/profile.php?updated=1` because that's when we sync from wp-admin to Calypso. We don't want the syncs to happen at the same time and step on each other. But, if it's a pedestrian GET request to `/wp-admin/profile.php` then we want to allow Calypso to wp-admin syncing.

Reply:
Sorry you're right; I was confused! 🙏